docker/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+pid_file='/var/run/gunicorn/gunicorn.pid'
+
+function stop_container {
+    service nginx stop
+    gunicorn_pid=$(cat ${pid_file})
+    kill -TERM $(cat ${pid_file})
+}
+
+trap stop_container SIGTERM
+
+echo 'Installed handler' 1>&2
+
+service nginx start
+
+su -c ". /home/mahiru/.env/bin/activate && gunicorn --pid ${pid_file} --access-logfile /var/log/gunicorn/access.log --error-logfile /var/log/gunicorn/error.log --capture-output --bind 127.0.0.1:8000 'proof_of_concept.rest.registry:wsgi_app()'" mahiru &
```

Comment:
Why do you have to activate the venv in this script? That is done by the docker image right?

Reply:
Activating a venv changes environment variables. Those are not stored in a Docker layer, so you have to activate it here again.